lms/static/styles/components/_BookPicker.scss

```diff
@@ -0,0 +1,12 @@
+.BookPicker {
+  // The containing `Modal` component will take up 90vw on narrow screens.
+  // For viewports that are wider, `Modal` sets a min-width, but it's a little
+  // narrow for this particular interface. The book-selector feels about right
+  // at 36em where space allows. 42em minimum here allows for at least 3em
+  // of horizontal breathing room on each side of the modal's contents.
+  width: min(90vw, 42rem);
```

Comment:
CSS `min` requires [a more recent browser](https://developer.mozilla.org/en-US/docs/Web/CSS/min()) than our current baselines. Assuming we update the baselines per [this issue](https://github.com/hypothesis/client/issues/3744) soon then I think its only older versions of Chrome where this might be an issue.

Is there a reasonable fallback we can set in the interim here (eg. add `width: 90vw` on the line above)?

Reply:
It's a little magical, admittedly, but the applied (shared) `Modal` styles that these rules extend already have that fallback.